pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -10,10 +10,21 @@ import (
 	f "github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
-func newMemcpyContinueCopyingHint(continueCopying hinter.ResOperander) hinter.Hinter {
+func newMemContinueHint(continueTarget hinter.ResOperander, memset bool) hinter.Hinter {
+	var name string
+	if memset {
+		name = "MemsetContinueLoop"
+	} else {
+		name = "MemcpyContinueCopying"
+	}
 	return &GenericZeroHinter{
-		Name: "MemcpyContinueCopying",
+		Name: name,
 		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			// MemsetContinueLoop
+			//> n -= 1
+			//> ids.continue_loop = 1 if n > 0 else 0
+
```

Comment:
i think the formatting standard here is to add `//>` also for empty line

Reply:
We do add that if the empty line belongs to the code of the hint, but do not think that's necessary here where you want to create an actual separation.
The reason we use `//>` instead of simple comment lines is because the hint code could be confused in some cases with actual go code from the vm and so it doesn't appear to be just commented code from the codebase

Reply:
Oh yeah you re right my bad :p